src/hub/serializers.py

```diff
@@ -164,3 +165,22 @@ def get_editor_permission_groups(self, hub_instance):
             context=context,
             many=True,
         ).data
+
+    def get_relevancy_score(self, hub_instance):
+        from researchhub_document.models import UnifiedDocumentConcepts
+
+        unified_document = self.context.get("unified_document", None)
+        relevancy_score = 0
+
+        if unified_document and hub_instance.concept:
```

Comment:
```python
if unified_document and concept := getattr(hub_instance, "concept", None):

```

Reply:
I think `hub_instance.concept` will throw an error if it happens to be null. This is because it's an 1-1 relationship, compared to a foreign key, which would usually return `None`.

Reply:
Seems like my version of python doesn't have the walrus operator even though I'm in 3.8. But, I will make the change (without the walrus)